zk-sdk/src/sigma_proofs/batched_grouped_ciphertext_validity/handles_2.rs

```diff
@@ -58,20 +59,26 @@ impl BatchedGroupedCiphertext2HandlesValidityProof {
     /// The function simply batches the input openings and invokes the standard grouped ciphertext
     /// validity proof constructor.
     ///
-    /// The function does *not* hash the public keys, commitment, or decryption handles into the
-    /// transcript. For security, the caller (the main protocol) should hash these public
-    /// components prior to invoking this constructor.
-    ///
     /// This function is randomized. It uses `OsRng` internally to generate random scalars.
+    #[allow(clippy::too_many_arguments)]
```

Comment:
Consider marking with `pub(crate)`. 
Indeed, the folder `sigma_proofs` is private at the crate level. 
However, this protects against future changes, signals intent and making the (auto generated) docs more clear. 

Also, it makes it consistent with `pub(crate) new_direct()` and `pub(crate) verify_direct()`.

Same for all `new()` and `verify()` functions in the `sigma_proofs` folder.